fw/http_msg.c

```diff
@@ -1492,36 +1492,57 @@ tfw_http_msg_expand_from_pool_lc(TfwHttpTransIter *mit, TfwPool* pool,
 	return __tfw_http_msg_expand_from_pool(mit, pool, str, tfw_cstrtolower);
 }
 
-static void
-__tfw_h2_msg_shrink_frags(TfwMsgIter *it, const char *body,
-			  TfwHttpRespCleanup *cleanup)
+static inline void
+__tfw_h2_msg_move_frags(struct sk_buff *skb, int frag_idx,
+			TfwHttpRespCleanup *cleanup)
 {
-	struct skb_shared_info *si = skb_shinfo(it->skb);
-	skb_frag_t *frag = &si->frags[it->frag];
-	char* frag_addr = skb_frag_address(frag);
-	int len;
 	int i;
+	int len;
```

Comment:
Why to keep 2 int variables in separate lines?  The same for `__tfw_h2_msg_rm_all_frags()`